scripts/install-chef-suse.sh

```diff
@@ -304,31 +304,6 @@ case "$rootpw" in
         ;;
 esac
 
-# It is exceedingly important that 'hostname -f' actually returns an FQDN!
-# if it doesn't, add an entry to /etc/hosts, e.g.:
-#    192.168.124.10 cb-admin.example.com cb-admin
-if ! FQDN=$(hostname -f 2>/dev/null); then
-    die "Unable to detect fully-qualified hostname. Aborting."
-fi
-
-if ! DOMAIN=$(hostname -d 2>/dev/null); then
-    die "Unable to detect DNS domain name. Aborting."
-fi
-
-if [ -z "$FQDN" -o -z "$DOMAIN" ]; then
-    die "Unable to detect fully-qualified hostname. Aborting."
-fi
-
-if ! resolved=$(getent ahosts $FQDN 2>/dev/null); then
-    die "Unable to resolve hostname $FQDN via host(1). Please check your configuration of DNS, hostname, and /etc/hosts. Aborting."
-fi
-
-IPv4_addr=$( echo "$resolved" | awk '{ if ($1 !~ /:/) { print $1; exit } }' )
-IPv6_addr=$( echo "$resolved" | awk '{ if ($1  ~ /:/) { print $1; exit } }' )
```

Comment:
-1: all the variables that were defined are actually used later in the script, so we need to keep them defined...